config/initializers/slack_notifier.rb

```diff
@@ -0,0 +1,11 @@
+require 'slack-notifier'
+
+if ENV['SLACK_WEB_HOOK_URL'] && ENV['SLACK_CHANNEL'] && ENV['SLACK_USERNAME']
```

Comment:
Maybe just add handling for production only env and for the tests you can construct a specific double/mock/stub in the spec helper?

return unless Rails.env.production?

Reply:
not sure, in that case we should probably move that to the `/environments/production.rb`

Reply:
This is the same approach you've taken elsewhere, though https://github.com/trade-tariff/trade-tariff-backend/commit/052c310553c107a7ebf5381e0532ac3c1f487df0

Reply:
I think when you're configuring a specific gem regardless of how we want it behave the approach you've taken elsewhere is the correct one.